docker/README.md

```diff
@@ -39,14 +39,34 @@ $ git clone git@github.com:openfoodfoundation/openfoodnetwork.git
 ```sh
 $ cd openfoodnetwork
 ```
-* Download the Docker images, build the Docker containers, seed the database with sample data, AND log the screen output from these tasks:
+* You have two choices of images you can build. The default Ubuntu image is most similar to the production servers but is larger in size and will take longer to build. Alternatively, you can use the Alpine image which is smaller, faster to setup, and should be compatible with the Ubuntu. To download the Ubuntu Docker images, build the Docker containers, seed the database with sample data, AND log the screen output from these tasks:
```

Comment:
Should we add a note here that the Alpine image will need updating when we change the Ruby version in the future?

And I think it's safe to say, we don't have the resources to officially support both Ubuntu and Alpine, so we would not necessarily be supporting Alpine.

Reply:
I added a note just to be safe. When the time comes it will be a pretty simple PR, but it could cause issues at the time.